app/views/components/branded-header/README.md

```diff
@@ -0,0 +1,17 @@
+Use the MyHMCTS Branded Header component for any service or system not hosted on GOV.UK such as those for the judiciary or HMCTS staff.
+
+Do not use this for citizen-facing services.
+
+{{dsExample({
+  name: 'branded-header',
+  example: 'default',
+  height: 150
+})}}
+
+## How it works
+
+MyHMCTS is displayed on the 'Left', with the name of the service or system appearing off-centered left. You can also add global links in the top right. For example, ‘Profile’ and ‘Sign out’.
```

Comment:
It only seems to be on the left when the screen size is small - as you make the screen wider, it starts moving across - not clear if this is the intention or not but I expected it to fix to the left.

Reply:
it currently sit right on the left of hmcts-header__container.
So when it's a big/wide screen it seems to have some margin left, but actually it's stay exactly on the left of the hmcts-header__container.